backend/wmg/data/schemas/cube_schema.py

```diff
@@ -42,13 +71,26 @@
     ]
 )
 
+expression_summary_fmg_domain = tiledb.Domain(
+    [
+        tiledb.Dim(name=cube_indexed_dim, domain=None, tile=None, dtype="ascii", filters=filters)
+        for cube_indexed_dim in expression_summary_fmg_indexed_dims
+    ]
+)
+
 # The cube attributes that comprise the core data stored within the cube.
 expression_summary_logical_attrs = [
-    tiledb.Attr(name="n_cells", dtype=np.uint32, filters=filters),
     tiledb.Attr(name="nnz", dtype=np.uint64, filters=filters),  # TODO: Why uint64?
     tiledb.Attr(name="sum", dtype=np.float32, filters=filters),
 ]
 
+expression_summary_fmg_logical_attrs = [
+    tiledb.Attr(name="sum", dtype=np.float32, filters=filters),
+    tiledb.Attr(name="sqsum", dtype=np.float32, filters=filters),
+    tiledb.Attr(name="nnz", dtype=np.uint64, filters=filters),
+    tiledb.Attr(name="nnz_thr", dtype=np.uint64, filters=filters),
+]
+
```

Comment:
could we place all `fmg` schema-related together? possibly in a new schema file? might allow us to remove the `fmg` infix and name the module something like `marker_genes_cube_schema.py` for clarity

Reply:
If you move them to a new file, also copy over some of the comments that explain what the variables are for.